### DISCUSSION TITLE
Current Observation API and Legacy Time format

### DISCUSSION THREAD

**Discussion by vinolys** [weather-gov/api#744] [General]
There is a discreancy between the "observation_time_rfc822" between the legacy XML and API XML downloads.
<observation_time_rfc822>Fri, 12 Jul 2024 10:52:00 -0500</observation_time_rfc822>   ----->  legacy  link 
 https://forecast.weather.gov/xml/current_obs/XXXX.xml
<observation_time_rfc822>Fri, 12 Jul 24 15:53:00 +0000</observation_time_rfc822>   --------> outout from API
It will be nice there is some consistancy in the time format.

**Comment by dwhitemv25** [comment]
It's the same time format as specified by RFC 5322, just different time zones. 

The date & time handling library for your programming language of choice should be able to deal with the necessary conversions.

**Edit**: I just noticed the API is using the obsolete RFC822 2-digit year format. It really should be using the RFC 5322 4-digit year. That is a real problem and should be fixed.
